src/shared/server-config.toit

```diff
@@ -91,8 +91,11 @@ class ServerConfigSupabase extends ServerConfig implements supabase.ServerConfig
   root-certificate-der/ByteArray? := ?
 
   constructor.from-json name/string json/Map [--der-deserializer]:
-    root-der-id := json.get "root_certificate_der_id"
-    root-der/ByteArray? := root-der-id and (der-deserializer.call root-der-id)
+    root-der-base64/string? := json.get "root_certificate_der"
```

Comment:
Maybe I'm getting a bit ahead of myself here, but I'd prefer downloading the server config as standard JSON (instead of UBJSON) to make it easier to reason about the config when pushed to a recovery server.

Reply:
I think we made it such that root-certificates could be shared between the Artemis server and a broker. That's why we have an ID, and then look up the DER this way.
Even if we don't need the Artemis server certificate anymore, it might still be nice to have the certificate separate, so it isn't in the middle of other configurations.

I think we can use JSON, and just have the certificates serialized as base64. The der-serializer just needs to deal with that.

Reply:
That's still the plan. But there is no id the der-deserializer can look up. So the new code tries to read an embedded (base64) der and if that isn't there, we get an id and deserialize that.

Not sure I fully understand your comment.